ruby/loops.md

```diff
@@ -0,0 +1,38 @@
+# Loops
+Now that we have gotten our hands a little dirty with ruby, let's discuss the
+concept of loops. Loops allow a program to execute particular tasks as
+many times as needed to achieve a particular goal. This is important because
+thousands of steps within a program can be completed within seconds, increasing
+productivity and reducing the time and effort that would be required if a user
+had to do it manually. Throughout this lesson you will learn about looping constructs, flow control and conditionals and why they are important concepts in computer programming.
+
+## Learning outcomes
+*Look through these now and then use them to test yourself after doing the assignment*
+
+After completing this lesson you can expect to:
+
+* Understand what a loop is and why its useful
+* Know how to use a `while` loop in ruby
+* Know how to use a `for` loop in ruby
+* Know how to use the `loop` loop in ruby
+* Know how use a `times` loop in ruby.
+* Know how to use the `until` loop in ruby.
+* Know how to use the `upto` and `downto` loops in ruby
+
+## Assignment
+1. Read through launchschool's chapter on [Loops and Iterators]
+(https://launchschool.com/books/ruby/read/loops_iterators#simpleloop).
+This chapter gives an excellent overview of 'while', 'until', 'do/while',
+and the 'for' loops. For an extra challenge, do the Exercises section at the end
+ of the chapter.
+2. Read [zetcode's](http://zetcode.com/lang/rubytutorial/flowcontrol/) article covering control flow. It is a good overview going over loops and explaining how to control the flow of Ruby programs by using what's known as conditionals.
+3. Check out lead Viking Erik Trautman's post, [Ruby Explained: Iteration](http://www.eriktrautman.com/posts/ruby-explained-iteration).
+
+## Exercise
+Link to external repo which will include exercises for this lesson which the user will complete by passing tests
+
+## Further Reading
+*This section contains helpful links to other content. It isn't required, so consider it supplemental for if you need to dive deeper into something*
+* [A Wealth of Ruby Loops and Iterators](http://www.skorks.com/2009/09/a-wealth-of-ruby-loops-and-iterators/)
+* [The Bastards Book of Ruby](http://ruby.bastardsbook.com/chapters/loops/)
+* [Introduction to Programming](https://www.teamten.com/lawrence/programming/intro/)
```

Comment:
This is a great beginner resource, it uses python in its examples though which has deceptively similar syntax to Ruby 😞 I think this might cause confusion but would like to hear your thoughts on including it 😄